src/uwtools/resources/jsonschema/ww3.jsonschema

```diff
@@ -11,8 +11,14 @@
             },
             "template_values": {
               "additionalProperties": {
-                "type": "string"
```

Comment:
Noticed this was hardcoded to string in prior revisions to remove the namelist subschema and fixed here